### ISSUE TITLE
Wireshark removed the manuf file

### ISSUE THREAD

**Issue by westnet-paul** [coolbho3k/manuf#34]
Looks like the wireshark project has replaced the manuf file with a .c file containing static arrays.

I'll try to come up with a pull request to fix this. It might take me a while.

**Comment by stavlor** [comment]
https://www.wireshark.org/download/automated/data/manuf has the data i think needed for this.

**Comment by solsticedhiver** [comment]
yes. Just change the MANUF_URL in manuf.py with that url, and it's good to go.
